### DISCUSSION TITLE
Impacts from Docker's announcement regarding end of free tier for organizations?

### DISCUSSION THREAD

**Discussion by capi** [TriliumNext/Trilium#3726] [General]
Docker recently announced that it will sunset and delete all free organizational accounts. https://github.com/docker/hub-feedback/issues/2314 provides some more discussions and opinions on that.

Does this affect the images published for Trilium or is this a personal account? Do you plan to publish Docker images also to e.g. [Github's container registry][1]?

Thanks for the cool project and I really appreciate the ready-to-use Docker images!

[1]: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry

**Comment by zadam** [comment]
I have a "user" account only, not an organization:

<img width="554" alt="image" src="https://user-images.githubusercontent.com/617641/225732308-d33339b5-3755-422a-b931-060414b7a1e3.png">
